airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -176,6 +189,58 @@ def _doc_upload(
     return doc_uploaded, doc_blob_id
 
 
+def _file_upload(
+    local_path: Path,
+    gcp_connector_dir: str,
+    bucket: storage.bucket.Bucket,
+    *,
+    upload_as_version: str | Literal[False],
+    upload_as_latest: bool,
```

Comment:
Nit: should this also have a default?

Reply:
Actually, neither have a default right now - this is to ensure an arg isn't forgotten. Since this is code we control and only is for us, I sacrificed some convenience in favor of keeping things explicit.